components/rhs_root_post/rhs_root_post.jsx

```diff
@@ -380,7 +413,7 @@ export default class RhsRootPost extends React.PureComponent {
         }
 
         let dotMenuContainer;
-        if (!isPostDeleted && this.props.post.type !== Constants.PostTypes.FAKE_PARENT_DELETED) {
+        if ((!isPostDeleted && this.props.post.type !== Constants.PostTypes.FAKE_PARENT_DELETED) && (this.state.hover || this.state.dropdownOpened || this.state.showEmojiPicker || this.state.a11yActive)) {
```

Comment:
Line too long. Maybe should we put the second condition (the or chain) in a variable, so it is also more clear what this second condition is representing?

Reply:
I couldn't think of a name so i put it in next line :P